pkgs/by-name/tt/tt/package.nix

```
@@ -0,0 +1,28 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec{
+  pname = "tt";
+  version = "0.4.2";
+
+  src = fetchFromGitHub {
+    owner = "lemnos";
+    repo = "tt";
+    rev = "refs/tags/v${version}";
+    sha256 = "oe0gegMINVmbw1UPvSy9G67D4amQTumlrgvb8H+Y4IM=";
+  };
+
+  vendorHash = "sha256-iny0OKInHqoXzYcEtd4f0T/yty5/z3k3yFbDo5yazes=";
+
+  postInstall = ''
+    mv $out/bin/src $out/bin/tt
+  '';
+
+  meta = with lib; {
+    description = "A typing test in the terminal written in Go";
```

Indefinite articles should be removed from the description:
```suggestion
    description = "Typing test in the terminal written in Go";
```